Additional quality metrics (#1981)
SpikeInterface/spikeinterface: doc/modules/qualitymetrics/firing_range.rst

Firing range (:code:`firing_range`)
===================================

Calculation
-----------

The firing range indicates the dispersion of the firing rate of a unit across the recording. It is computed by

> taking the difference between the 95-th and 5th percentiles firing rates computed over short time bins (e.g. 10 s).

Reviewer: ```suggestion
taking the difference between the 95th percentile's firing rate and the 5th percentile's firing rate computed over short time bins (e.g. 10 s).
```

Reviewer: I think making this slightly more explicit will read cleaner. Otherwise it technically needs a percentiles' apostrophe for a plural possessive. Which is easier to confuse a reader.

Expectation and use
-------------------

> Very high levels of firing ranges, outside of a physiolocigal range, might indicate noise contamination.

Reviewer: ```suggestion
Very high levels of firing ranges, outside of a physiological range, might indicate noise contamination.
```

Example code
------------

.. code-block:: python

    import spikeinterface.qualitymetrics as qm

    # Make recording, sorting and wvf_extractor object for your data.
    firing_range = qm.compute_firing_ranges(wvf_extractor)

>     # firing_range is a dict containing the units' IDs as keys,

Reviewer: ```suggestion
    # firing_range is a dict containing the unit IDs as keys,
```

I think the meaning is that each key is a different unit ID, in which case you would say unit IDs. The way it is written means that each unit has multiple IDs that are being used as keys, but maybe I'm misunderstanding what data is being returned?

Author: done for all quality metrics in last commit
